Akanksha7-art

Akanksha7-art opened the pull request.

akanksha-max committed:
Akanksha7-art

sudeepshukla338 commented:
Hey @Akanksha7-art ,
Can you list out , what are the things you have updated . if possible , please include image of updated web page, so I can get an idea , how its look.. So that I can merge it with main branch.
Thankyou :)

Akanksha7-art commented:
Thank You for reaching out.
The Problem I Saw -  The issue I felt, with the website is that the colors
used in the site were very dark, and harsh on the reader's eye.
I changed - I updated the "index.html " file. I have given a better (as for
me) css to the site. It is more reader-friendly now.
I am attaching the new look of the site.


Thank you, hope to get a positive response from your end.

On Tue, 6 Oct 2020 at 22:15, sudeepshukla338 <notifications@github.com>
wrote:

> Hey @Akanksha7-art <https://github.com/Akanksha7-art> ,
> Can you list out , what are the things you have updated . if possible ,
> please include image of updated web page, so I can get an idea , how its
> look.. So that I can merge it with main branch.
> Thankyou :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sudeepshukla338/poetry_website/pull/2#issuecomment-704407022>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APZRZSRW26WSEWOFGWEPEH3SJNCTDANCNFSM4SFPYNPA>
> .
>

Akanksha7-art committed:
Merge branch 'master' into master

Akanksha7-art commented:
Is there any issue with my PR, will it be merged?
